libs/src/api/account.js

```diff
@@ -145,22 +148,26 @@ class Account extends Base {
       // through signup
       if (createWAudioUserBank && this.solanaWeb3Manager) {
         phase = phases.SOLANA_USER_BANK_CREATION
-        try {
-          // Fire and forget createUserBank. In the case of failure, we will
-          // retry to create user banks in a later session before usage
-          (async () => {
+        // Fire and forget createUserBank. In the case of failure, we will
+        // retry to create user banks in a later session before usage
+        (async () => {
+          try {
+            handleUserBankOutcomes('Create User Bank: Request')
```

Comment:
Don't love having these just passed in as strings. Could we pass in constants (`USER_BANK_CREATION_STEP.REQUEST`?), and then export those to the client so it can handle them as it pleases?